modules/wordads/css/style.css

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
It might be good to add that CSS to the concatenated `jetpack.css`, like so:
f2fd0513646664440a293d6c42f64d1f21d73d02